timescaledb/getting-started/compress-data.md

```diff
@@ -1,6 +1,6 @@
 # 7. Create a compression policy
```

Comment:
Didn't notice before that we have two of the same headings on this page. Maybe...

```suggestion
# 7. Compression policies
```